classes/trait.ilObjShowDuplicates.php

```diff
@@ -162,10 +161,9 @@ public function handleMultiReferences(int $a_obj_id, int $a_ref_id, string $a_fo
         $il_access_handler = $this->ilias_dic->access();
 
         // process
-
         /** @var ObjectSettings $objectSettings */
         $objectSettings = ObjectSettings::find($a_obj_id);
-        if ($all_refs = $objectSettings->getDuplicatesOnSystem()) {
+        if ($objectSettings && $all_refs = $objectSettings->getDuplicatesOnSystem()) {
```

Comment:
I would not recommend using a variable and fetching a variable in a same if statement, I would recommend doing it like:
```suggestion
        $all_refs = $objectSettings ?  $objectSettings->getDuplicatesOnSystem() : null;
        if (!empty($all_refs)) {
```